### PR TITLE
Adding Hamburger Menu

### PR DESCRIPTION
**Pull request by solomonetchu**
This Pull Request was made for creating a dynamic   Hamburger Menu in The Mobile Version.
Also, My Coding partner was not here, I have made repeated communication but to no avail.

### CODE REVIEW

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @solomonetchu,** 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## **Invalid Code Review Request**

You have submitted a project, that _**Please add a description to your PR to make the PR more Pro, then re-submit for another review.**_

Capture web_11-10-2022_10324_github com

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by solomonetchu** [comment]
This Pull Request is for the Mobile Hamburger Menu; This was made Dynamic with JavaScript; using the DOM (Vanilla Js)

**Review by GzimAsani — CHANGES_REQUESTED** [review]
Hi @solomonetchu, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have a PR description, kindly add one. You can update it here
image


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: Script/index.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+const hamburger = document.querySelector(".hmburger");
+const navMenu = document.querySelector(".nav__mobile");
```
Comment (AmaduKamara) [inline Script/index.js:2]: - [ ] I noticed that the classes you are selecting here do not exist in the HTML file, or at least do not exist in the HTML file for this branch. And for that reason, when I click on the hamburger menu, nothing works. Kindly work on having these classes and their contents to make the Mobile menu work as expected before your project can be ready for approval 👍 
- [ ] Also, the style of the hamburger menu you are having does not match the original design given. Please work on the styles to match your design with the template chosen and also consider the Welcome text (this is supposed to be your name as the logo) style to match the original one as well. 👍 


**Your Header Menu**
image


**The Original Design for Header and Hamburger Menu**
image

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️♻️♻️ 

Hi @solomonetchu,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✅ You used Gitflow correctly
✅ Linter checks passed
✅ Good commit messages 👍 

## Required Changes ♻️
_Check the comments under the review._

- [ ] Kindly work on the style of the Mobile Header to match the original design given so that when one clicks on the hamburger menu, the mobile menu appears and all functionalities work as required in your project requirement. 👍 I trust you can fix that.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by AnselemOdims — APPROVED** [review]
### Hi @solomonetchu 🙋‍♀️, 
Good job in making the required changes 👏 

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Use the tag @AnselemOdims for a quicker reply


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**